modules/payloads/python/shellcode_inject/aes_encrypt_HTTPKEY_Request.py

```diff
@@ -0,0 +1,456 @@
+# encoding=utf8
+"""
+
+This payload has AES encrypted shellcode stored on a webserver.  At runtime, the executable
+uses the key from a HTML requet holding the key, using md5 to hash the html output and produce 
+required 16 Byte key. Than injects it into memory, and executes it.
+
+[*] Logic:
+- HTTP Request on your hosted webpage:                                     <-----
+    DOWN ----> Sleep and try to reach supplied webpage using HTTP response code |
+    UP ----> Preform MD5 hash of HTML of webpage:
+        MD5 ---> pass to decryption function:
+            CalL Shellcode ----> Shellcode invojes callback: WIN 
+
+
+[*] Benefits:
+- Uses urllib2 to make get request against supplied server and returns the 
+  HTML text for md5 hashing.
+- Prevents Key being deployed with payload, preventing payload to run in sandbox
+  if target server is taken offline during initial deployment.
+- Once use of payload is over, take down webserver to prevent future infections
+- Static and future dynamic RE is near impossible without proper data collection. 
+- Proper web log monitoring will identify when webserver is burnt and crucial to 
+  remove key to prevent future ability to RE binary. Defenders would need to capture 
+  live memory to capture key.
+- Devloped custom HTML login page to be hosted as key for payload
+
+
+
+---------------------------------------
+Based off AES encrypt module by @christruncer 
+
+module by @KillSwitch-GUI
+
+
+"""
+
+import urllib2
+import md5
+
+from datetime import date
+from datetime import timedelta
```

Comment:
Doesn't look like the above imports are used, can you confirm?  If not, go ahead and remove them